parquet-column/src/main/java/org/apache/parquet/column/ParquetProperties.java

```diff
@@ -55,6 +57,16 @@
  */
 public class ParquetProperties {
```

Comment:
Since this class defines `ParquetProperties.WriterVersion` that is exposed in the public API, I think it would be a breaking change to rename it. This class is otherwise not part of the public API (and I've tried to keep it that way here) so we should fix this when we separate the API into well-defined packages.